moto/core/exceptions.py

```diff
@@ -95,6 +95,12 @@ def to_json(self) -> "JsonRESTError":
         err.code = self.code
         return err
 
+    @classmethod
+    def extended_templates(cls, extended_templates: Dict) -> Dict:
```

Comment:
Perhaps I should refactor this to include the DictLoader Environment boilerplate as well